src/Pim/Component/Catalog/Factory/ProductValue/ProductValueFactoryInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace Pim\Component\Catalog\Factory\ProductValue;
+
+use Pim\Component\Catalog\Model\AttributeInterface;
+use Pim\Component\Catalog\Model\ProductValueInterface;
+
+/**
+ * Factory that creates product values.
+ *
+ * @internal  Interface for the factories used internally by \Pim\Component\Catalog\Factory\ProductValueFactory.
```

Comment:
Does it make sense to tag this interface as internal? It can be confusing since we want integrators to be able to write their own implementations if needed.